### ISSUE TITLE
Квест "Экзорцизм над полковником Джулсом"

### ISSUE THREAD

**Issue by XandrKhv** [wowmw/bugtracker#457]
Квест берется у Ассистента Клату в Оплоте Чести
При юзании Ритуальных молитвенных четок ничего не происходит
квест не проходится

**Comment by kuzinsun** [comment]
Приветсвую!

Четки нужны для лечения NPC https://db.moonwell.su/?npc=22431 во время ритуала - работает.
Квест работает корректно, выполняется.

Спасибо за обращение.

**Comment by XandrKhv** [comment]
> Приветсвую!
> 
> Четки нужны для лечения NPC https://db.moonwell.su/?npc=22431 во время ритуала - работает. Квест работает корректно, выполняется.
> 
> Спасибо за обращение.

Ну не работает он
https://youtu.be/BHzT7-CYxUA

**Comment by kuzinsun** [comment]
> > Приветсвую!
> > Четки нужны для лечения NPC https://db.moonwell.su/?npc=22431 во время ритуала - работает. Квест работает корректно, выполняется.
> > Спасибо за обращение.
> 
> Ну не работает он https://youtu.be/BHzT7-CYxUA

Доброго времени суток!

Спасибо за предоставленное видео.

Есть подозрение, что зависает ивент после некорректного запуска предыдущим игроком. Перепроверим - ответим.

**Comment by kuzinsun** [comment]
@XandrKhv 
![image](https://user-images.githubusercontent.com/60097141/212569434-eb137089-41ac-4b8d-bbc7-cbbd78879679.png)
Есть диалог с этим NPC https://db.moonwell.su/?npc=22431 ? Ивент запускается у него.

**Comment by XandrKhv** [comment]
> @XandrKhv ![image](https://user-images.githubusercontent.com/60097141/212569434-eb137089-41ac-4b8d-bbc7-cbbd78879679.png) Есть диалог с этим NPC https://db.moonwell.su/?npc=22431 ? Ивент запускается у него.

Да, после рестарта сервера мне удалось с ним поговорить. Но смогли ли его выполнить после меня, это вопрос)

**Comment by XandrKhv** [comment]
> @XandrKhv ![image](https://user-images.githubusercontent.com/60097141/212569434-eb137089-41ac-4b8d-bbc7-cbbd78879679.png) Есть диалог с этим NPC https://db.moonwell.su/?npc=22431 ? Ивент запускается у него.

вот видео, на котором видно что диалога тоже нет
https://www.youtube.com/watch?v=9AIcpY4qYJA
